src/components/login/login.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField } from '@material-ui/core';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+interface State {
+  APIClient: rest.Evergreen;
+  open : boolean;
+  submitted : boolean
+  username : string
+  password : string
+}
+
+class Props {
+}
+
+export class Login extends React.Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.handleClickOpen = this.handleClickOpen.bind(this);
+    this.handleClickClose = this.handleClickClose.bind(this);
+    this.handleClickSubmit = this.handleClickSubmit.bind(this);
+    this.state = {
+      APIClient: rest.EvergreenClient("admin", "e4f2c40463dcade5248d36434cb93bac", "http://localhost:8080/api"),
+      open : false,
+      submitted : false,
+      username : "",
+      password : "",
+    };
+  }
+
+  public handleClickOpen() : any{
+    this.setState({
+      open : true,
+    });
+  }
+
+  public handleClickClose(): any {
+    this.setState({
+      open : false,
+    });
+  }
+
+  public handleClickSubmit(): any {
+    console.log(this.state.username);
+    this.setState({
+      open : false,
+      submitted : true,
+    });
+  }
+
+  public render() {
+    return (
+      <div>
+        <Button onClick={this.handleClickOpen}>
+          Log In
+        </Button>
+        <Dialog open={this.state.open} onClose={this.handleClickClose} aria-labelledby="form-dialog-title">
```

Comment:
This isn't a big deal, but aria- properties aren't necessary for Evergreen code. They are used to annotate elements for accessibility software such as screen readers, but accessibility is not one of our goals

Reply:
Good to know. The Dialog example I found in the material-ui documentation included these properties, and I wasn't totally sure what they did so I just left them alone. I have now removed them.